django_ses/__init__.py

```diff
@@ -138,6 +138,15 @@ def send_messages(self, email_messages):
         source = settings.AWS_SES_FROM_EMAIL
         email_feedback = settings.AWS_SES_RETURN_PATH
         for message in email_messages:
+            if settings.AWS_SES_USE_BLACKLIST:
+                from django_ses import utils
+                message.to = utils.filter_blacklisted_recipients(message.to)
+                message.cc = utils.filter_blacklisted_recipients(message.cc)
+                message.bcc = utils.filter_blacklisted_recipients(message.bcc)
+
+                if len(message.to) + len(message.cc) + len(message.bcc) == 0:
+                    return
```

Comment:
Should we put a logger debug in here that can be used to see if messages get filtered out?

Reply:
Yes! Sounds like a good idea!